origin.spec

```diff
@@ -51,7 +51,7 @@ Release:        0%{?dist}
 Summary:        Open Source Container Management by Red Hat
 License:        ASL 2.0
 URL:            https://%{import_path}
-ExclusiveArch:  x86_64
+ExclusiveArch:  x86_64 ppc64le
```

Comment:
This isn't really "platform independent".  If we do this, let's really make it platform independent.
Something like this would be good.

# If go_arches not defined fall through to implicit golang archs
%if 0%{?go_arches:1}
  ExclusiveArch:  %{go_arches}
%else
  ExclusiveArch:   x86_64 aarch64 ppc64le 
%endif